src/Apps/Artwork/Components/PrivateArtwork/PrivateArtworkAboutWork.tsx

```diff
@@ -32,9 +33,8 @@ export const PrivateArtworkAboutWork: React.FC<PrivateArtworkAboutWorkProps> = (
       <HTML variant="lg">
         <ReadMore
           inlineReadMoreLink={false}
-          content={
-            "Through a practice that is self-described as monastic, austere and concrete, Balliano’s meticulous paintings appear, upon first glance, clean and precise. However, closer inspection reveals scrapes and scratches that uncover the organic wooden surface underneath the layers of paint, as a decaying façade of modernistic intentions"
-          }
+          content={`${data.artwork?.additionalInformation}`}
```

Comment:
I'm getting null when trying this with a few artworks on staging when using both `bio` and `additionalInformation` (line 16) which may be because those artworks are missing some info.

I should probably add some more fake data on staging cms to see which field actually retrieves this text? Otherwise the syntax is probably incorrect somewhere.